mne/report.py

```diff
@@ -652,14 +674,22 @@ class Report(object):
         Subject name.
     title : str
         Title of the report.
+    cov_fname : str
+        Name of the file containing the noise covariance.
+    baseline : str | bool
+        If True, apply baseline. If 'auto', baseline correct only if highpass
+        filter has not been applied or is less than 1.0 Hz.
```

Comment:
@agramfort any better idea how to avoid the baseline mess by default? At least this is magic for a good purpose.